docs/readmes/resources/ref_magma_metrics.md

```diff
@@ -0,0 +1,186 @@
+---
+id: ref_magma_metrics
+title: Life of a Magma Metric
+hide_title: true
+---
+
+## Life of a Magma Metric
+
+This document will cover the life of a Magma metric as it originates at the
```

Comment:
Formatting nit: *don't* break the lines at 79 chars -- use one paragraph per line, as if you were writing in e.g. google docs

Other norms: https://magma.github.io/magma/docs/next/contributing/contribute_conventions#markdown

Reply:
```suggestion
This document will cover the life of a metric in a Magma deployment as it originates at the
```

Reply:
Note: AGW metrics are just one source of metrics. Any gateway can report its own metrics (e.g. CWAG, FeG, etc), Orc8r can originate its own metrics, and Orc8r can scrape metrics from other targets (e.g. AWS or Postgres metrics)

That said, I think your focus (AGW -> Orc8r -> NMS) is still the right way to explain this, but I think adding some context in the intro about what I said above would be helpful for readers